netbout-web/src/test/java/com/netbout/rest/bout/TkStartTest.java

```diff
@@ -0,0 +1,115 @@
+ * OF THE USE OF THIS SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF
+ * SUCH DAMAGE.
+ */
+package com.netbout.rest.bout;
+
+import com.jcabi.urn.URN;
+import com.netbout.mock.MkBase;
+import com.netbout.rest.RqWithTester;
+import com.netbout.rest.TkStart;
+import com.netbout.spi.Alias;
+import com.netbout.spi.Bout;
+import com.netbout.spi.Message;
+import com.netbout.spi.User;
+import java.util.regex.Matcher;
+import java.util.regex.Pattern;
+import org.hamcrest.MatcherAssert;
+import org.hamcrest.Matchers;
+import org.junit.Ignore;
+import org.junit.Test;
+import org.takes.facets.forward.TkForward;
+import org.takes.rq.RqFake;
+import org.takes.rs.RsPrint;
+
+/**
+ * Test case for {@link TkStart}.
+ * @author Ivan Inozemtsev (ivan.inozemtsev@gmail.com)
+ * @version $Id$
+ * @since 2.15
+ */
+public final class TkStartTest {
+
+    /**
+     * TkStart can post message and invite friends to
+     * newly created bouts.
+     * @throws Exception If there is some problem inside
+     */
+    @Test
+    public void postsMessageAndInvitesFriends() throws Exception {
+        final MkBase base = new MkBase();
+        final URN urn = new URN("urn:test:1");
+        final User user = base.user(urn);
+        final String jeff = "Jeff";
+        user.aliases().add(jeff);
+        final String frank = "Frank";
+        final User friend = base.user(new URN("urn:test:2"));
+        friend.aliases().add(frank);
+        final String post = "message";
+        final String head = new RsPrint(
+            new TkForward(new TkStart(base)).act(
+                new RqWithTester(
+                    urn,
+                    new RqFake(
+                        "GET",
+                        String.format(
+                            "/start?post=%s&invite=%s",
+                            post,
+                            frank
+                        )
+                    )
+                )
+            )
+        ).printHead();
+        final Matcher matcher = Pattern.compile(
+            "^Location:.*/b/(\\d+)$",
+            Pattern.MULTILINE
+        ).matcher(head);
+        MatcherAssert.assertThat("Location header is correct", matcher.find());
+        final long number = Long.parseLong(matcher.group(1));
+        final Alias alias = user.aliases().iterate().iterator().next();
+        final Bout bout = alias.inbox().bout(number);
+        final Message message = bout.messages().iterate().iterator().next();
+        MatcherAssert.assertThat(message.author(), Matchers.equalTo(jeff));
+        MatcherAssert.assertThat(message.text(), Matchers.equalTo(post));
+        MatcherAssert.assertThat(
+            bout.friends().iterate().iterator().next().alias(),
+            Matchers.equalTo(frank)
+        );
+    }
+
+    /**
+     * TkStart can remove all friends from a bout when some of invitees
+     * is not found.
+     * @throws Exception If there is some problem inside
+     * @todo #610 negative test is not implemented
```

Comment:
@komaz let's put the TODO in the class javadoc and remove this empty method. Also, TODOs should start with `#ISSUE_NUMBER:TIME_INmin` and have at least 20 words, or pdd will complain.